### PR TITLE
lang: [ar] update strings

### PR DESCRIPTION
**Pull request by mshannaq**
**Description**
Supersedes #398

Add missing Arabic Translation files and strings.

### CODE REVIEW

File: Language/ar/Core.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
     'copyError'                    => 'حدث خطأ أثناء محاولة استبدال الملف. يرجى التأكد من أن المجلد قابل للكتابة.',
     'enabledZlibOutputCompression' => 'إن قيمة الموجه zlib.output_compression فعالة في ملف ini. هذا لن يعمل بشكل جيد مع مخازن الإخراج المؤقتة.',
     'invalidFile'                  => 'ملف غير صالح: {0}',
+    'invalidDirectory'             => 'المجلد غير موجود: "{0}"',
```
Comment (kenjis) [inline Language/ar/Core.php:17]: Only one `"`. Is it okay?

**Comment by mshannaq** [comment]
If you are taking about invalidDirectory

it is 2 "  and 2 '
you can check the string

+    'invalidDirectory'             => 'المجلد غير موجود: "{0}"',

copy the string into note pad and try to make space

+    'invalidDirectory'             => 'المجلد غير موجود: "{0}" ',




*M*ohammed  Al*S*hannaq
Linux Servers Administrator & Senior Backend Web developer

Mobile Phone (Jordan) (WhatsApp) : +962-77-777-4221

Personal E-Mail: ***@***.***
LinkedIn: https://www.linkedin.com/in/mshannaq/
Personal Website: http://ms.per.jo
Personal Blog (In Arabic): http://irbidnet.com/blog/
Youtube Channel (In Arabic):
https://www.youtube.com/channel/UC4PoeBJ2tM3fzLNBYXllx4g
Twitter: @mshannaq <http://twitter.com/mshannaq>


Note: This message may contain information that is legally privileged or
confidential. If you have received this communication in error, please
notify the Mohammed AlShannaq immediately by e-mail reply and delete it
from your system, as dissemination of this communication is strictly
prohibited.
Since the integrity and security of this message cannot be fully guaranteed
on the Internet, we strongly advise that you understand and observe this
lack of security when e-mailing us.

**Comment by mshannaq** [comment]
@kenjis I can remove the " " If it will cause a visual issue, but the php is correct , it just a visual issue as arabic is RTL language.

do you want me to remove " double quotation mark in "{0}"?

**Comment by kenjis** [comment]
@mshannaq No. I don't know Arabic, so I have no opinion.

**Review by MGatner — APPROVED** [review]

**Review by kenjis — APPROVED** [review]

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**